docs/1.0.0/install_ubuntu.md

```diff
@@ -0,0 +1,258 @@
+
+To install python 3.7-dev run the following command:
+
+```shell script
+sudo apt-get install libpq-dev python3.7-dev
+```
+
+### 3. Install `pip`
+
+If you didn't update your local APT repository, execute:
+
+```shell script
+sudo apt update
+```
+
+Install `pip3` by executing:
+
+```shell script
+sudo apt-get install python3-pip
+```
+
+### 4. Install `virtualenv`
+
+If you didn't update your local APT repository, execute:
+
+```shell script
+sudo apt update
+```
+
+Install `virtualenv` by executing:
+
+```shell script
+sudo apt-get install virtualenv
+```
+
+### 5. Install Lumino Invoicing dependencies
+
+Lumino includes a billing that is based on the Lighting Network invoicing feature. In order to install Lumino, the following dependencies are required:
+
+```shell script
+sudo apt install libsecp256k1-dev
+
+sudo apt-get install libssl-dev build-essential automake pkg-config libtool libffi-dev libgmp-dev libyaml-cpp-dev
+```
+
+## Build RIF Lumino from code
+
+### 1. Get the code
+
+Get the code from [the Github page](https://github.com/rsksmart/lumino/). You can either clone the repo or get the compressed version from the [releases page](https://github.com/rsksmart/lumino/releases). 
+
+Let's call your local path in which the code resides `$RIF_LUMINO_PATH`.
+
+### 2. Create an environment
+
+You'll only need to create a python virtual environment for RIF Lumino once. 
+
+Execute the following command:
+
+```shell script
+virtualenv -p <PATH_TO_PYTHON3.7> clientEnv
+```
+
+**Note:**
+
+Replace `<PATH_TO_PYTHON3.7>` with the path where Python3.7 is installed in your system. In the case of Ubuntu OS, this is usually `/usr/bin/python3.7`.
+
+You can verify your path to Python3.7 by executing:
+```shell script
+which python3
+```
+
+### 3. Activate the environment
+Activate the python virtual environment by executing the following command:
+
+```shell script
+source clientEnv/bin/activate
+```
+
+Check if the Python version is correct inside the virtual environment by running:
+
+```shell script
+python --version
+```
+
+This command should output version 3.7.x.
+
+### 4. Install RIF Lumino requirements in your environment
+
+Inside the virtual environment run the following command (this could take a few minutes):
+
+```shell script
+pip install -r requirements.txt -c constraints.txt -e .
+```
+
+Run the Lumino setup with the following command:
+
+```shell script
+python setup.py develop
+```
+
+## Start RIF Communications transport layer
+
+The comunication between Lumino nodes can be done both using RIF Communications (https://www.rifos.org/communications) and Matrix (https://matrix.org/).
+
+The default way, and the one we encourage to use for a more decentralized ecosystem, is RIF Communications. 
+
+In order to run Lumino using RIF Comms, you need to set up the RIF Communications bootnode. To use Matrix, no configuration is required. 
+
+### Set up a RIF Communications bootnode
+
+#### Requirements:
+
+- Node 14+ (14.13 recommended)
```

Comment:
@mauricioirace can you confirm whether we recommend `14.13` or `14.15`?

Reply:
14.15.4 is the one in production, so I'd go for 14.15. (anyway both works fine)